static/js/main.js

```diff
@@ -0,0 +1,135 @@
+/**
+ * Function that fetches the recent edits by a user.
+ *
+ * 'encodeURIComponent' function has been used to escape unsafe characters in the URL.
+ * Similarly, unsafe characters have been escaped in HTML also.
+ * This will prevent injection in HTML and URL.
+ */
+function fetchRecentEdits() {
+  //Displaying 'Fetching Results' message till the actual results get loaded.
+  document.getElementById("results").innerHTML =

```

Comment:
You could use jQuery here as well. Not that there is anything wrong with this code, it's just harder to read.